frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Alert, AlertActionLink, Button, Skeleton } from '@patternfly/react-core';
+import {
+  LOG_REFRESH_RATE,
+  LOG_TAIL_LINES,
+} from '~/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/const';
+
+type LogsTabStatusProps = {
+  error?: Error;
+  loaded: boolean;
+  refresh: () => void;
+  onDownload: () => void;
+  canDownload: boolean;
+};
+
+const LogsTabStatus: React.FC<LogsTabStatusProps> = ({
+  error,
+  loaded,
+  refresh,
+  onDownload,
+  canDownload,
+}) => {
+  if (error) {
+    return (
+      <Alert
+        isInline
+        variant="danger"
+        title="An error occurred while retrieving the requested logs"
+        actionLinks={<AlertActionLink onClick={refresh}>Retry</AlertActionLink>}
+      >
+        <p>{error.message}</p>
```

Comment:
The error alert looks great 🥳 
<img width="707" alt="Screenshot 2023-08-31 at 11 25 36 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/c0cc91c6-1af1-47f1-b0f5-157e8ede9bcd">

Future scope: Can we provide more specific error messages? Like specific messages for "skipped nodes" and "old pipelines".